test/scripts/v1beta1/create-cluster.sh

```diff
@@ -14,26 +14,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This shell script is used to build a cluster and create a namespace from our
-# argo workflow
+# This shell script is used to create a EKS cluster.
 
 set -o errexit
 set -o nounset
 set -o pipefail
 
 CLUSTER_NAME="${CLUSTER_NAME}"
-ZONE="${GCP_ZONE}"
-PROJECT="${GCP_PROJECT}"
-NAMESPACE="${DEPLOY_NAMESPACE}"
+AWS_REGION="${AWS_REGION}"
 
-echo "Activating service-account"
-gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
-echo "Creating CPU cluster"
-gcloud --project ${PROJECT} beta container clusters create ${CLUSTER_NAME} \
-    --zone ${ZONE} \
-    --machine-type=n1-standard-8 \
-    --num-nodes=6 \
-    --cluster-version 1.16
-echo "Configuring kubectl"
-gcloud --project ${PROJECT} container clusters get-credentials ${CLUSTER_NAME} \
-    --zone ${ZONE}
+echo "Starting to create EKS cluster"
+
+eksctl create cluster \
+    --name ${CLUSTER_NAME} \
+    --version 1.17 \
+    --region ${AWS_REGION} \
+    --zones us-west-2a,us-west-2b,us-west-2c \
+    --nodegroup-name linux-nodes \
+    --node-type m5.xlarge \
+    --nodes 2 \
+    --nodes-min 1 \
+    --nodes-max 3
```

Comment:
Can we use managed nodegroup? 

Check here: https://github.com/kubeflow/testing/blob/master/images/aws-scripts/create-eks-cluster.sh#L36

Reply:
I think I will use your script to create EKS cluster: https://github.com/kubeflow/testing/blob/master/images/aws-scripts/create-eks-cluster.sh.
What do you think @PatrickXYS ?

Reply:
> I think I will use your script to create EKS cluster
Yes we can, reuse common script makes a lot of sense to me.